### PR TITLE
[luci] Quantize bias shared by multiple nodes.

### PR DESCRIPTION
**Pull request by jinevening**
This enables to quantize bias shared by multiple nodes.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
@jinevening , can you separate the pass that clones shared bias?
- quantize can have multiple phase.  first phase can do these kind of cloning to make things simple.
- second phase will do the quantization.

**Comment by jinevening** [comment]
@seanshpark I've updated the PR not to clone a FLOAT32 bias. Instead, it directly creates a quantized bias (INT32 or INT64) from the FLOAT32 bias.

This would be the first work for https://github.com/Samsung/ONE/issues/5785.

File: compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp [hunk 1]
```diff
@@ -503,8 +553,8 @@ struct QuantizeActivation final : public luci::CircleNodeMutableVisitor<bool>
         continue;
 
       // Check if this is bias (bias is quantized later)
```
Comment (seanshpark) [inline compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:555]: Maybe need to change this comment too

File: compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp [hunk 2]
```diff
@@ -563,64 +613,77 @@ struct QuantizeBias final : public luci::CircleNodeMutableVisitor<bool>
       return false;
 
     // Check if this is bias
```
Comment (seanshpark) [inline compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:615]: maybe update this comment too

File: compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp [hunk 3]
```diff
@@ -563,64 +613,77 @@ struct QuantizeBias final : public luci::CircleNodeMutableVisitor<bool>
...
-    auto iw = get_input_weight_of_bias(node);
-    if (iw.first == nullptr || iw.second == nullptr)
-      return false;
-
-    auto input = loco::must_cast<luci::CircleNode *>(iw.first);
-    auto weight = loco::must_cast<luci::CircleNode *>(iw.second);
+    auto iwo_list = get_input_weight_output_of_bias(node);
 
-    if (granularity == QuantizationGranularity::ChannelWise)
+    for (auto iwo : iwo_list)
```
Comment (seanshpark) [inline compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:618]: From this line to the end, it seems change is not that much but the whole code shows diff.
Can you please use `// clang-format off` to reduce diff and `TODO: fix indentation` or an appropriate word?

File: compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp (hunk 1 above)
Comment (jinevening) [reply compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:555]: This comment does not have to be changed.

File: compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp (hunk 2 above)
Comment (jinevening) [reply compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp]: I've changed this comment.

File: compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp (hunk 3 above)
Comment (jinevening) [reply compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:619]: I've added clang-format off and on. It makes the PR much clearer. PTAL :)

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**